config/routes/overrides.rb

```diff
@@ -29,6 +29,7 @@ def matches?(request)
     member do
       get :pulp_storage
       get :pulp_status
+      get :capsule_puppet_path
```

Comment:
I'm also not exactly sure, what this does, I more added it because it seemed like everything that was being used with functions alongside capsule_puppet_path, so I had figured it was needed. I'll try it without and see if it doesn't need to be changed.